examples/example-gateway/build/clients/bar/bar.go

```diff
@@ -208,6 +208,10 @@ func NewClient(
 	port := deps.Default.Config.MustGetInt("clients.bar.port")
 	baseURL := fmt.Sprintf("http://%s:%d", ip, port)
 	timeout := time.Duration(deps.Default.Config.MustGetInt("clients.bar.timeout")) * time.Millisecond
+	defaultHeaders := make(map[string]string)
+	if deps.Default.Config.ContainsKey("clients.bar.defaultHeaders") {
```

Comment:
why not always generate `defaultHeaders` empty map in config by default? :)

Reply:
config file is not necessarily generated. From zanzibar perspective, we assume config files will be read/written by humans, in which sense less clutter is good.

Reply:
make sense, for edge-gateway tho is always gen-ed from ui